packages/smooth_app/lib/pages/product/confirm_and_upload_picture.dart

```diff
@@ -73,24 +73,35 @@ class _ConfirmAndUploadPictureState extends State<ConfirmAndUploadPicture> {
                           });
                           retakenPhoto.delete();
                         }),
-                    SmoothActionButton(
-                      text: _getConfirmButtonText(
-                        context,
-                        widget.imageType,
+                    const SizedBox(width: 10),
+                    Flexible(
+                      child: SizedBox(
+                        child: Tooltip(
+                          message: _getConfirmButtonText(
+                            context,
+                            widget.imageType,
+                          ),
+                          child: SmoothActionButton(
+                            text: _getConfirmButtonText(
```

Comment:
It will always be "Confirm" from now on.

Reply:
I have created a confirm_button_label in app_en.arb file. How can I create it in other .arb files

Reply:
it's propagated automatically by our translation system once the PR is merged @Gagarod